src/navigation/index.js

```diff
@@ -24,25 +25,41 @@ const AppNavigator = createStackNavigator(
               alignItems: 'center',
               padding: 10
             }}
-            onPress={() => {}}
             activeOpacity={0.5}
           >
-            <Icon name="search" color={colors({}).secondary} size={25} />
+            <Icon name="search" color={colors().secondary} size={25} />
           </TouchableOpacity>
         )
       }
     },
     Profile: {
-      screen: ProfileScreen,
-      navigationOptions: ({ navigation }) => {
-        const { params: { username } = {} } = navigation.state;
-        return {
-          title: `${username}'s profile`,
-          headerStyle: {
-            backgroundColor: colors({}).primary
-          },
-          headerTintColor: colors({}).secondary
-        };
+      screen: ProfileScreen
+      // navigationOptions: ({ navigation }) => {
```

Comment:
Hi @Musigwa 
It is good practice to delete code you are not using in your codebase rather than commenting it out.